sqlglot/dialects/sqlite.py

```diff
@@ -65,6 +65,7 @@ class Generator(generator.Generator):
 
         TRANSFORMS = {
             **generator.Generator.TRANSFORMS,  # type: ignore
+            **transforms.ELIMINATE_DISTINCT_ON,  # type: ignore
```

Comment:
this is not correct, you've munged eliminate_qualify since they're both keyed on select, you need to do custom here

exp.Select: preprocess([eliminate_distinct_on, eliminate_qualify])

can you delete all transform constants in transform.py and replace them with this more explicit version?